Creategoalpanel4mobile

franarave opened the pull request:
https://github.com/DuckyTeam/ducky-web/issues/1462

franarave committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into CreateGoalPanel4Mobile

franarave committed:
CreateGoalPanel4Mobile

https://github.com/DuckyTeam/ducky-web/issues/1462

franarave committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into CreateGoalPanel4Mobile

nathan-bird commented on src/CreateGoalPanel4Mobile/index.js:
```diff
@@ -0,0 +1,96 @@
+        size={"hr2"}
+        />
+      <CreateGoalActionItem className={styles.actionItemComposite}
+        co2={props.co2}
```
These `CreateGoalActionItem`components would be passed in by the parent since the number and type of activity isn't fixed. You can create a new function that expects a type of object then go through that object to make the number of action items.

franarave replied:
Done!

nathan-bird reviewed (changes requested):
Looks good, but lets add a bit more logic to the component :)

franarave committed:
CreateGoalActionItem

New function that expects a type of object then go through that object
to make the number of action items created-

nathan-bird committed:
added to json
